inlong-manager/manager-test/src/main/resources/sql/apache_inlong_manager.sql

```diff
@@ -151,7 +151,7 @@ CREATE TABLE `third_party_cluster`
     `id`          int(11)      NOT NULL AUTO_INCREMENT COMMENT 'Incremental primary key',
     `name`        varchar(128) NOT NULL COMMENT 'Cluster name',
     `type`        varchar(32)  NOT NULL COMMENT 'Cluster type, including TUBE, PULSAR, etc.',
-    `ip`          varchar(64)  NOT NULL COMMENT 'Cluster IP',
+    `ip`          text         NOT NULL COMMENT 'Cluster IP',
```

Comment:
please, change this line comment, and give one example.

Reply:
OK, I changed it, if the cluster name was empty, query data proxy cluster by type and select the first one to return.